docs/developer-guide/build-service.md

```diff
@@ -84,53 +84,13 @@ Repeat this process for the footer:
 
 The build service is capable of including more than one component in the same bundle, so you can simply add multiple modules into the same URL.  Here's an example:
 
-	<link rel="stylesheet" href="//build.origami.ft.com/bundles/css?modules=o-header@^3.0.0,o-footer@^3.0.0" />
+	<link rel="stylesheet" href="//build.origami.ft.com/bundles/css?modules=o-fonts@^1,o-ft-icons@^2,o-header@^3.0.0,o-footer@^3.0.0" />
 
 It's important that you do this, so that any CSS that is shared between the header and footer (there's quite a bit) isn't downloaded twice.
 
 <aside>Remember that when you change the modules you are requesting in your build service tags, it may take a few minutes to build the resulting bundle of code.  Be patient and hit 'Run with JS' a few times until the styling appears.</aside>
 
 
-### Loading fonts and icons
-
-Edit the `<link>` tag to load styles for these modules (you'll need to look them up in the registry as you did for header and footer):
-
-* `o-ft-icons`
-* `o-fonts`
-
-You should have something like this:
-
-	<link rel="stylesheet"
-	      href="//build.origami.ft.com/bundles/css?modules=o-header@^3.0.0,o-footer@^3.0.0,o-fonts@^1.4.0,o-ft-icons@^2.1.1" />
```

Comment:
Good point. A bit too random. Although some people in the workshop tended to skip that bit completely and seemed to dislike that they had to work for something rather trivial to appear correctly.

How about adding something in the comments of the CTM example like that?

> we've loaded o-fonts and o-ft-icons for you, so that the text looks on brand and icons appear correctly